src/meta/src/backup_restore/restore.rs

```diff
@@ -146,31 +165,46 @@ async fn restore_impl(
             target_id
         )));
     }
+    let mut target_snapshot = backup_store.get(target_id).await?;
+    tracing::info!(
```

Comment:
dumb question: will info log be printed to stdout?

Reply:
Yes, when using the restore binary, risingwave logger has been properly set [here](https://github.com/risingwavelabs/risingwave/blob/c4516dea1366d9df06ccf7e28ef2bdabf6fb1019/src/storage/backup/cmd/src/bin/backup_restore.rs#L28).